octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/util/OpenDetailTabEntityMouseListener.java

```diff
@@ -61,7 +61,8 @@ public void mouseClick(EntityModel entityModel, MouseEvent event) {
 		            Entity.FEATURE,
 		            Entity.AUTOMATED_TEST,
 		            Entity.AUTOMATED_TEST_RUN,
-		            Entity.BDD_SCENARIO));
+		            Entity.BDD_SCENARIO,
+					Entity.REQUIREMENT_FOLDER));
```

Comment:
the spacing needs to be corrected

Reply:
I checked it in notepad++ and eclipse and it looks good. Idk why it is like this in git